lib/fluent/plugin/out_s3.rb

```diff
@@ -97,6 +97,8 @@ def initialize
     config_param :enable_dual_stack, :bool, default: false
     desc "If false, the certificate of endpoint will not be verified"
     config_param :ssl_verify_peer, :bool, :default => true
+    desc "Full path to the SSL certificate authority bundle file that should be used when verifying peer certificates. If unspecified, defaults to the system CA if available."
```

Comment:
Could you add this document under [here](https://github.com/fluent/fluent-plugin-s3/blob/e1b206d5d0837bcdd55636f46ffe9bd76b423d3e/docs/output.md#ssl_verify_peer) too?

Reply:
Updated